Configuration/Settings.yaml

```diff
@@ -106,9 +106,18 @@ Flowpack:
 Neos:
   ContentRepository:
     Search:
+      # API. If set to FALSE, only index the "live" workspace and not user workspaces.
+      # If you only index the live workspace, Search will not work for your editors in the user workspaces.
+      # Furthermore, if you heavily rely on Search for collecting content, this might be strange for editors to
+      # work with -- as unpublished changes are not indexed right away.
       indexAllWorkspaces: true
+
       elasticSearch:
-        indexName: typo3cr
+
+        # API. name of the Elasticsearch index to use. Will create many indices prefixed by this indexName.
+        indexName: neoscr
```

Comment:
Does the (new) PR description work for you? I'd also make sure a release hints at the needed changes. Or should we stay with the "old name" to be on the safe side?

And… didn't we change that name elsewhere already anyways? Hm…